docker/orchestrator/build.sh

```diff
@@ -5,14 +5,14 @@ set -e
 tmpdir=`mktemp -d`
 
 script="go get github.com/youtube/vitess/go/cmd/vtctlclient && \
-  git clone https://github.com/enisoc/orchestrator.git src/github.com/outbrain/orchestrator && \
-  go install github.com/outbrain/orchestrator/go/cmd/orchestrator"
+  git clone https://github.com/enisoc/orchestrator.git src/github.com/github/orchestrator && \
```

Comment:
You should be able to switch the clone from my fork to the upstream repo. I only used my fork here temporarily while some changes needed for Vitess were still in PR phase.